.gitignore

```diff
@@ -79,4 +79,4 @@ Network Trash Folder
 Temporary Items
 .apdisk
 
-
+/guestbook/db.sqlite3
```

Comment:
Try to keep the paths in here relative to the repository. You can use the `.gitignore` file in your user's home directory for user-global settings.